SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialVirtualWorkerTranslator.h

```diff
@@ -56,6 +58,7 @@ class SPATIALGDK_API SpatialVirtualWorkerTranslator
 
 	// The WorkerId of this worker, for logging purposes.
 	FString WorkerId;
+	VirtualWorkerId LocalVirtualWorkerId;
```

Comment:
As I understand, `SpatialVirtualWorkerTranslator::LocalVirtualWorkerId` must be equal to `UAbstractLBStrategy::LocalVirtualWorkerId`. If that's the case - could we remove one of those to avoid potential mismatches?

Reply:
Possibly, but then the UAbstractLBStrategy would need to know about the SpatialVirtualWorkerTranslator. Maybe that is fine?

Reply:
This was added specifically to remove that circular reference that we had before. It's the only piece of information in the Translator that the Strategy needs so this was the only way I could get the Strategy tests to work without also providing a Translator.

Reply:
Makes sense, let's keep both then.